Library/ViewModels/MessageBannerViewModel.swift

```diff
@@ -53,14 +54,34 @@ MessageBannerViewModelInputs, MessageBannerViewModelOutputs {
     self.messageTextAlignment = bannerType.map { $0.textAlignment }
 
     self.bannerMessage = self.messageBannerConfiguration.signal.skipNil().map(second)
+    self.bannerMessageAccessibilityLabel = self.bannerMessage
+      // todo: Needs to be localized - Strings.banner_tap_to_dismiss
+      .map { "\($0), double tap to dismiss" }
 
```

Comment:
should "double tap to dismiss" be a hint?

Reply:
I tried using `accessibilityHint` but VoiceOver would simply not read it. I've seen this happen before, that VoiceOver would in certain situations not read the `hint` but don't really have a clue as to why.

Thinking about this a bit more led me to Apple's documentation - https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint where you can clearly read: "Avoid beginning the phrase with the imperative form of a verb, because this can make the hint sound like a command. For example, do not create a hint such as “Play the song” or “Purchase the item." which made me extend the `accessibilityLabel` instead.

I think in this situation using an `accessibilityLabel` works slightly better by using comma (there is a small pause) - if we were to use a `hint` (and it in fact worked) the pause would be considerably bigger.

Reply:
Given that the sentence is using an imperative form verb and that VoiceOver is not reading the hint I think this is a great work around. Thanks for explaining!